### ISSUE TITLE
 Undefined property: CsnUser\Entity\User::$displayName in C:\wamp\www\ZD3\vendor\coolcsn\csn-user\src\CsnUser\Entity\User.php on line 321

### ISSUE THREAD

**Issue by abszar** [coolcsn/CsnUser#31]
Hi i installed CsnUser more than one time and when i Edit a user Profile it update but this error keep showing on top  "Undefined property: CsnUser\Entity\User::$displayName in C:\wamp\www\ZD3\vendor\coolcsn\csn-user\src\CsnUser\Entity\User.php on line 321"

**Comment by wingman007** [comment]
I will fix the problem today and push again. SO in few hours you can do composer update and you will get the updated version.

**Comment by abszar** [comment]
i dont know if it help but i change the 321 line on User.php to this : 
    public function getDisplayName()
    {
        if (!empty($this->displayName)) {
            return $this->displayName;
        } return '';
    }

and the message did not appear again

**Comment by wingman007** [comment]
Yes, this is a quick fix. Thank you for sharing! I will do the same in the repository just to get rid of the warning for now and will make more changes later.
